spec/lib/dfe_signin/utils_spec.rb

```diff
@@ -0,0 +1,33 @@
+require "rails_helper"
+
+RSpec.describe DfeSignin::Utils do
+  before do
+    DfeSignin.configure do |config|
```

Comment:
Again, can we rely on the config being already set and test against those values rather than re-configuring during the spec?